app/views/layouts/_breadcrumbs.html.haml

```diff
@@ -1,5 +1,5 @@
 - if @breadcrumbs && @breadcrumbs.length > 1
-  %ol.breadcrumb
+  %ol.breadcrumb{:style => "display: none;"}
```

Comment:
This needs to go away before merging, right?

Reply:
This hides the old breadcrumbs, because the new and old breadcrumbs should not be visible at the same time. And this is only a temporary solution: the old breadcrumbs should be removed completely in another PR.